static/app/views/settings/project/projectSettingsNavigation.tsx

```diff
@@ -12,9 +12,15 @@ type Props = {
 };
 
 const ProjectSettingsNavigation = ({organization, project}: Props) => {
-  const appStoreConnectContext = useContext(AppStoreConnectContext);
+  const {promptItunesSession, appstoreCredentialsValid} = useContext(
+    AppStoreConnectContext
+  ) || {
+    promptItunesSession: false,
+    // Default to not showing any alerts if we have no info
+    appstoreCredentialsValid: true,
```

Comment:
Not so sure about this, but for now you can extract on the `promptItunesSession` part and not worry about the semantics of this I think :)

Reply:
By only checking `promptItunesSession` this would change the original behaviour though, to be fair: The original check would be true if either of these are true, as `updateAlertMessage` is initialized to an object containing some user-facing text based on whether they needed to update their credentials or refresh their iTunes session.